packages/legend-application-studio/src/components/editor/edit-panel/project-configuration-editor/ProjectConfigurationEditor.tsx

```diff
@@ -360,13 +430,97 @@ export const ProjectConfigurationEditor = observer(() => {
   };
   const disableAddButton =
     selectedTab !== CONFIGURATION_EDITOR_TAB.PROJECT_DEPENDENCIES || isReadOnly;
-  const updateConfigs = (): void => {
-    editorStore.localChangesState.alertUnsavedChanges((): void => {
-      flowResult(configState.updateConfigs()).catch(
-        applicationStore.alertUnhandledError,
+
+  const updateGavConfigs = (): void => {
+    const documentationEntry =
+      applicationStore.documentationService.getDocEntry(
+        LEGEND_STUDIO_DOCUMENTATION_KEY.QUESTION_HOW_TO_DEFINE_A_CONSTRAINT,
       );
+
+    const actions = [
+      {
+        label: (
+          <>
+            Acknowledge and Proceed
+            {documentationEntry && (
```

Comment:
why don't just use the `DocumentationLink` component?

Reply:
It's for the `event.preventDefault();` line, since I wasn't sure how we could pass in an event within the `DocumentationLink` component